src/components/Home.css

```diff
@@ -0,0 +1,109 @@
+@import url('https://fonts.googleapis.com/css2?family=Gothic+A1:wght@100;200;300;400;500;600;700;800;900&family=Lato:wght@100;300;400;700&family=Montserrat:wght@100;200;300;400;500;600&family=Permanent+Marker&family=Poppins:ital,wght@0,100;0,300;0,400;0,500;0,600;1,100&family=Space+Grotesk:wght@300;400;500;600;700&family=Titan+One&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  font-family: 'Poppins', sans-serif;
+  color: white;
+}
+
+body {
+  background-image: url('../features/maroon.jpg');
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+  background-attachment: fixed;
+}
+
+input[type=text] {
+  width: 75%;
+  padding: 10px 15px;
+  margin: 20px 20px;
+  box-sizing: border-box;
+  font-size: 18px;
+}
+
+.title {
+  padding-top: 20px;
+  font-size: 20px;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+}
+
+.angle-arrow {
+  color: white;
+  font-size: 20px;
+  position: relative;
+  left: 45%;
+}
+
+.cards {
+  list-style: none;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+.card {
+  padding: 8px;
+  width: calc(50% - 24px);
+  box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1);
+  background-color: rgba(202, 69, 111, 1);
+  display: inline-block;
+  vertical-align: top;
+}
+
+.card p {
+  padding: 1px 0;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+
+.details-link {
+  display: block;
+  text-align: right;
+  margin-top: 10px;
+  color: #007bff;
+  text-decoration: none;
+  font-weight: bold;
+}
+
+.details-link:hover {
+  text-decoration: underline;
+}
+
+.loading-spinner {
+  border: 4px solid rgba(0, 0, 0, 0.3);
+  border-top: 4px solid black;
+  border-radius: 50%;
+  width: 24px;
+  height: 24px;
+  animation: spin 1s linear infinite;
+  position: relative;
+  left: 50%;
+}
+
+@keyframes spin {
+  0% { transform: rotate(0deg); }
+  100% { transform: rotate(360deg); }
+}
+
+@media screen and (min-width: 768px) {
+  .card {
+    width: 320px;
+  }
+
+  input[type=text] {
+    width: 25%;
+    padding: 12px 20px;
+    margin: 20px 20px;
+    box-sizing: border-box;
+    font-size: 20px;
+  }

```

Comment:
- [ ] Good job in implementing the changes requested by the previous reviewer, :clap: , However, while making those changes, you might have made a mistake of giving your search textfield a font color, as it is right now, your font color is not visible in the search field. Please resolve that. :slightly_smiling_face: 
- Below is the screenshot of your search field when I typed `To`.
![Invisible](https://github.com/brighton0102/weather-app/assets/43528594/bf327e86-33bd-4c68-9fa4-d7453ca17d5e)